examples/siro_sandbox/sandbox_app.py

```diff
@@ -399,12 +400,12 @@ def _camera_pitch_and_yaw_wasd_control(self):
             self._lookat_offset_yaw += cam_rot_angle
 
     def _camera_pitch_and_yaw_mouse_control(self):
-        enable_mouse_control = self.gui_input.get_key(GuiInput.KeyNS.R) and (
-            (
-                self._first_person_mode
-                and self._cursor_style == Application.Cursor.HIDDEN_LOCKED
-            )
-            or (not self._first_person_mode)
+        enable_mouse_control = (
```

Comment:
For first-person mode, if the user left-clicks to hide the cursor, it makes sense to me to just always do mouse-look, without having to hold the R key.